### PR TITLE
log4j 2.17.1

### PR DESCRIPTION
**Pull request by TasDeniz**
use log4j version 2.17.1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
     <sonar.organization>samply</sonar.organization>
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-    <slf4jVersion>1.7.30</slf4jVersion>
```
Comment (patrickskowronekdkfz) [inline pom.xml:121]: Please look at [https://www.slf4j.org/manual.html](url). We want to use slf4j and log4j as the backend. In the future we can then decide if we want to change to logback or something else.

**Review by patrickskowronekdkfz — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (alexanderkiel) [reply pom.xml:121]: Please move the property to the position, the log4j version was before.

File: pom.xml [hunk 2]
```diff
@@ -260,7 +261,22 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <version>${slf4jVersion}</version>
+      <version>${slf4j.version}</version>
+    </dependency>
+    <dependency>
```
Comment (alexanderkiel) [inline pom.xml:266]: We should not need the log4j API.

File: pom.xml [hunk 3]
```diff
@@ -260,7 +261,22 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <version>${slf4jVersion}</version>
+      <version>${slf4j.version}</version>
```
Comment (alexanderkiel) [inline pom.xml:264]: We should not need the slf4j-simple implementation. API should be sufficient.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
I get a  java.lang.ClassNotFoundException: org.apache.logging.log4j.core.util.SetUtils at start.

Looks like we need the web dependency: https://stackoverflow.com/questions/70428495/issue-with-log4j-2-17-0-update-classnotfoundexception-setutils

**Review by alexanderkiel — APPROVED** [review]

**Review by patrickskowronekdkfz — APPROVED** [review]

**Pull request merged**